deploy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+# Defaults for production.
+# Example for development:
+#   PORT=8100 OPENFISCA_PORT=2000 OPENFISCA_CONFIG=development PUBLIC_URL=http://next.mes-aides.sgmap.fr ./deploy.sh aah
+TARGET_BRANCH=${1:-master}  # demo
+PORT=${PORT:-8000}  # 8100
+OPENFISCA_PORT=${OPENFISCA_PORT:-12000}  # 2000
+OPENFISCA_CONFIG=${OPENFISCA_CONFIG:-mes-aides}  # development
+PUBLIC_URL=${PUBLIC_URL:-https://mes-aides.gouv.fr}  # http://next.mes-aides.sgmap.fr
+
+# Install Mes Aides
+if ! cd mes-aides-ui
+then
+    git clone https://github.com/sgmap/mes-aides-ui.git
+    cd mes-aides-ui
+fi
+
+git fetch origin $TARGET_BRANCH
+git checkout origin/$TARGET_BRANCH
+
+rm -rf node_modules # work around libsass bindings being never found
+npm install
+grunt build
+
+# Stop Mes Aides
+killall --user `whoami` node || echo 'No server was running'
```

Comment:
Do you want this to be explored in this PR or delay it until creating a user for each instance is too annoying? Even if we solved this problem here, there would still be some manual work to set up multiple instances, mostly in setting up the reverse proxy and creating a new OpenFisca ini file.